etc/docker/docker_entrypoint.sh

```diff
@@ -96,4 +105,5 @@ while [[ ! -f /var/log/syslog ]]; do
     sleep 1
 done
 
-tail -f /var/log/syslog
+tail -f /var/log/syslog &
+wait $!
```

Comment:
is this change necessary?

Reply:
Yes, the script needs to be the one catching the sigterm from docker to execute the shutdown procedure. Without this, the child process will get it instead.

Reply:
thanks!